src/main/java/duke/command/ByeCommand.java

```diff
@@ -0,0 +1,35 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+
+/**
+ * This class executes the exit command
+ */
+public class ByeCommand implements Command {
+
+    /**
+     *
+     * @param taskList Manages all current tasks
+     * @param ui Used to print messages
+     * @param storage Loads and saves the tasks to a txt file
+     * @throws DukeException thrown if there are input/parsing errors
+     */
```

Comment:
noticed that most javadocs are not in a proper formatting. we have the same issue right here